openshift-ci/run_metallb_e2e.sh

```
@@ -51,15 +51,21 @@ export PATH=${PATH}:${HOME}/.local/bin
 export CONTAINER_RUNTIME=podman
 export RUN_FRR_CONTAINER_ON_HOST_NETWORK=true
 
-mkdir -p /tmp/report
+FAILED=0
+
+IBGP_REPORTER_PATH="${REPORTER_PATH}ibgp/"
+EBGP_REPORTER_PATH="${REPORTER_PATH}ebgp/"
+mkdir -p "${IBGP_REPORTER_PATH}" "${EBGP_REPORTER_PATH}"
+
 inv e2etest --kubeconfig=$(readlink -f ../../ocp/ostest/auth/kubeconfig) \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)